src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,36 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import User from '../../types/User';
+import { UserInfo } from '../UserInfo';
+import Todo from '../../types/Todo';
+import usersFromServer from '../../api/users';
+
+interface Props {
+  todo: Todo
+}
+
+export const TodoInfo: React.FC<Props> = ({
+  todo,
+}) => {
+  const {
+    id, title, completed, userId,
+  } = todo;
+
+  const user: User
+    = usersFromServer.find((us: User) => us.id === userId)
+    ?? usersFromServer[0];
```

Comment:
```suggestion
    = usersFromServer.find((us: User) => us.id === userId)
   || null;
```
it's strange to set the first user if we cannot find the real one in this array